scripts/o3de/o3de/engine_template.py

```diff
@@ -2331,15 +2331,16 @@ def create_repo(repo_path: pathlib.Path,
         logger.error('Remote repository URI cannot be empty.')
         return 1
     
-    # check if a repo.json file already exist in directory - can only have one repo.json file per project
     repo_json_path = repo_path / 'repo.json'
+    # if a repo.json file already exist in directory - can only have one repo.json file per project
     if not force and repo_json_path.is_file():
-        logger.error(f'{repo_json_path} already exists.  Use the --force to overwrite the existing file.')
+        logger.error(f'{repo_json_path} already exists. If you want to edit the repo.json properties, use edit-repo-properties command.'
+                     'Use --force if you want to override the current repo.json file')
         return 1
     elif repo_path.is_file():
         logger.error(f'{repo_path} already exists and is a file instead of a directory.')
         return 1
-    else:
+    elif not repo_path.is_dir():
```

Comment:
Why is this check needed? Also, if this check is done here, then the following call
`os.makedirs(repo_path, exist_ok=force)` won't need the `exist_ok` argument, since it will never exist.

Reply:
Hi Steve, the check is needed when user wants to create a folder at a path that already has a existed folder. If no check is performed then there will be a windows error with no appropriate error message: FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\o3de_Projects\\o3deRemoteRepo' and exist_ok = force to allow user to override that folder.

Reply:
To me, the original logic looks fine without this check, just as long as you do `exist_ok=True` as an argument to `os.makedirs` instead. `force` doesn't look like the appropriate value to set here because it is not actually forcing anything. If true, it will skip creating a folder if it already exists. If false, it will say 'you are creating a folder that already exists`. (see https://docs.python.org/3/library/os.html#os.makedirs )  The `force` argument should only apply to overriding any existing `repo.json` file.

Reply:
Thank you for the suggestion, this way would be more clear.